From: Eric Dorland <eric@debian.org>
Date: Sun, 20 Mar 2016 16:20:22 -0400
Subject: Use system help2man if it is available

---
 configure.ac     | 3 +++
 doc/Makefile.inc | 2 +-
 2 files changed, 4 insertions(+), 1 deletion(-)

Index: automake-1.16/configure.ac
===================================================================
--- automake-1.16.orig/configure.ac	2018-03-10 16:49:52.298353413 -0500
+++ automake-1.16/configure.ac	2018-03-10 16:49:52.270353739 -0500
@@ -102,6 +102,9 @@
 AC_CHECK_PROGS([YACC], [yacc byacc 'bison -y'], [false])
 AC_CHECK_PROGS([LEX], [lex flex], [false])
 
+# Use system help2man if its available
+AC_PATH_PROG([HELP2MAN], [help2man], ["$PERL \${srcdir}/doc/help2man"])
+
 # Test for Autoconf.  We run Autoconf in a subdirectory to ease
 # deletion of any files created (such as those added to
 # autom4te.cache).  We used to perform only the last of the three
Index: automake-1.16/doc/local.mk
===================================================================
--- automake-1.16.orig/doc/local.mk	2018-03-10 16:49:52.298353413 -0500
+++ automake-1.16/doc/local.mk	2018-03-10 16:49:52.270353739 -0500
@@ -37,7 +37,7 @@
 update_mans = \
   $(AM_V_GEN): \
     && $(MKDIR_P) %D% \
-    && ./pre-inst-env $(PERL) $(srcdir)/%D%/help2man --output=$@
+    && $(HELP2MAN) --output=$@
 
 %D%/aclocal.1 %D%/automake.1:
 	$(AM_V_GEN): \
